### ISSUE TITLE
The `require-accessible-name` rule is an error if only a Vue.js mutable value.

### ISSUE THREAD

**Issue by kimulaco** [markuplint/markuplint#469]
## About

The [`require-accessible-name`](https://markuplint.dev/rules/require-accessible-name) rule is an error if only a Vue.js mutable value.

It is not an error in React(JSX).

Do I need to disable the `require-accessible-name` rule if only a Vue.js mutable value?

**Vue Example**

```html
<!-- good -->
<h2>heading 1</h2>
<h2>heading {{count}}</h2>

<!-- bad -->
<h2>{{headingText}}</h2>
```

**React Example**

```html
<!-- good -->
<h2>heading 1</h2>
<h2>heading {{count}}</h2>
<h2>{{headingText}}</h2>
```

## Enviroment

- Sample repository: [kimulaco/markuplint-examples](https://github.com/kimulaco/markuplint-examples/tree/v0.1.0)
    - Directory: [examples/vue3](https://github.com/kimulaco/markuplint-examples/tree/v0.1.0/examples/vue3/)
        - [@components/HelloWorld.vue](https://github.com/kimulaco/markuplint-examples/tree/v0.1.0/examples/vue3/src/components/HelloWorld.vue#L3-L4)
- module version
    - Node.js v16.15.0
    - npm v8.5.5
    - markuplint v2.7.0
    - @markuplint/vue-parser v2.3.0
    - @markuplint/vue-spec v2.1.0

**Comment by YusukeHirao** [comment]
@kimulaco 

Thank you for the reporting.

It cannot evaluation the accessible name if an element has mutable content or attribute because the variable is may empty. So I'm puzzled that it successed to evaluate in React. Rather the behavior in Vue is as expected.

However I want to resolve that too. I think add an option that it evaluates even if the content or attribute is mutable.

Anyhow I just inspect your report. Please a moment.

**Comment by kimulaco** [comment]
Thank you.
I understand why it is an error.
I agree to add an option!
